controllers/provider-gcp/pkg/controller/worker/machines_test.go

```diff
@@ -283,7 +283,8 @@ var _ = Describe("Machines", func() {
 						"machineType": machineType,
 						"networkInterfaces": []map[string]interface{}{
 							{
-								"subnetwork": subnetName,
+								"subnetwork":        subnetName,
+								"disableExternalIP": false,
```

Comment:
Can you add a separate test case where `disableExternalIP` will be set to `true`?